src/rdf4cpp/rdf/IRIFactory.hpp

```diff
@@ -99,6 +90,8 @@ struct IRIFactory {
     [[nodiscard]] static nonstd::expected<IRI, IRIFactoryError> create_and_validate(std::string_view iri, storage::node::NodeStorage &storage = storage::node::NodeStorage::default_instance()) noexcept;
 
 private:
+    [[nodiscard]] std::string to_absolute(std::string_view rel) const noexcept;
```

Comment:
please double check if some methods here can be made static. (have to go now otherwise I would have checked myself)

Reply:
I've converted all functions where it is possible to internal linkage